src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsController.java

```diff
@@ -36,26 +37,43 @@
     private final BulkPrintService bulkPrintService;
     private final FeatureToggleService featureToggleService;
     private final AssignedToJudgeDocumentService assignedToJudgeDocumentService;
+    private final HelpWithFeesDocumentService helpWithFeesDocumentService;
 
     @PostMapping(value = "/case-orchestration/notify/hwf-successful", consumes = APPLICATION_JSON_VALUE)
-    @ApiOperation(value = "send e-mail for HWF Successful.")
+    @ApiOperation(value = "Notify Applicant/Applicant Solicitor of HWF Successful by email or letter.")
```

Comment:
do we ever notify applicant? or always applicant solicitor

Reply:
If its a paper application (via BSP) then it can be the applicant themselves without a solicitor